src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -1402,8 +1402,8 @@ CHIP_ERROR VerifyAttestationCertificateFormat(const ByteSpan & cert, Attestation
 
                 if (p != seqStart + len)
                 {
+                    // Failure to read will leave pathLen == -1
                     result = mbedtls_asn1_get_int(&p, end, &pathLen);
-                    VerifyOrExit(result == 0, error = CHIP_ERROR_INTERNAL);
```

Comment:
Is this line being removed the thing that fixes the linked issue?  Because I don't see how the change to treat pathLen == 0 as error would fix it, but maybe I am missing something?